js/client-battle.js

```diff
@@ -425,7 +425,7 @@
 					if (move.id === 'Struggle' || move.id === 'Recharge') pp = '&ndash;';
 					if (move.id === 'Recharge') move.type = '&ndash;';
 					if (name.substr(0, 12) === 'Hidden Power') name = 'Hidden Power';
-					var moveType = this.getMoveType(move, switchables[pos]);
+					var moveType = this.getMoveType(move, this.battle.mySide.active[pos]);
```

Comment:
`getMoveType` needs the active Pokémon to check whether it is under the effect of Embargo.